feature-wallet-impl/src/main/java/jp/co/soramitsu/wallet/impl/presentation/send/SendSharedState.kt

```diff
@@ -24,11 +24,16 @@ class SendSharedState {
     }
 
     fun updateAddress(address: String) {
+        clearAddress()
```

Comment:
do we need this here, before new value submitted?

Reply:
Yes because of MutableStateFlow